gcp/tsconfig.json

```diff
@@ -1,18 +1,9 @@
 {
+    "extends": "@tsconfig/node14/tsconfig.json",
     "compilerOptions": {
-        "strict": true,
+        "module": "esnext",
+        "target": "es2020",
+        "moduleResolution": "Node",
         "outDir": "bin",
-        "target": "es2016",
```

Comment:
same question as above on the TS compiler flags

Reply:
I will resolve this and we can continue the conversation there.